cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -241,6 +254,23 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	repoTmpl.Execute(w, data)
 }
 
+func (s *Server) updateOptions(w http.ResponseWriter, r *http.Request) {
+	if r.Method != "POST" {
```

Comment:
If you are going for full-on REST, should this be `OPTIONS`? or am I mistaken

Reply:
`OPTIONS` handlers return what method options are allowed for an endpoint. For example, if we supported the options method for this endpoint, it would just return `POST`. We browsers usually use this to make sure that the request it is about to send is supported.

Reply:
From https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/OPTIONS:

> The HTTP OPTIONS method is used to describe the communication options for the target resource. The client can specify a URL for the OPTIONS method, or an asterisk (*) to refer to the entire server.

I'm happy to change the name to settings rather than options to reduce confusion.